Improper documentation for installation and running

I believe that the instructions in the readme.md file is not correct.

cd dhp-api
npm i
**node start**

The node start command yields the following error
node start
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module 'C:\Users\Jeff\source\repos\dhp-api\start'
←[90m    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)←[39m
←[90m    at Module._load (node:internal/modules/cjs/loader:841:27)←[39m
←[90m    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)←[39m
←[90m    at node:internal/main/run_main_module:23:47←[39m {
  code: ←[32m'MODULE_NOT_FOUND'←[39m,
  requireStack: []
}

I believe this should be ...

cd dhp-api
npm i
**npm run start**

--- a/README.md
+++ b/README.md
@@ -56,7 +56,7 @@ Note: Environment variables must be set, as described in following sections, bef
 ```
 cd dhp-api
 npm i
-node start
+npm run start
 ```
 
 To execute all tests run the following from a command line.

